src/main/java/org/jenkinsci/plugins/github_branch_source/GitHubSCMNavigator.java

```diff
@@ -167,7 +164,11 @@ public String getApiUri() {
             for (GHRepository repo : user.listRepositories()) {
                 add(listener, observer, repo);
             }
+            return;
         }
+
+        listener.getLogger().format("%n%s does not correspond to a known GitHub User Account or Organization%n%n", repoOwner);
```

Comment:
:ant: redundant, delete. The `AbortException.message` is already displayed in the log.